diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/LineLength.kt

```diff
@@ -49,7 +50,7 @@ class LineLength : Rule("line-length") {
         var offset = 0
         node.text.lines().forEach {
             if (it.length > configuration.lineLength) {
-                val newNode = node.psi.findElementAt(offset + it.length - 1)!!.node
+                val newNode = node.psi.findElementAt(offset + it.trim().length - 1)!!.node
```

Comment:
Why do you need to trim here? I think this will return the same node, since you will point to effectively same line of code.

Reply:
No, it won't work without trim, because there are spaces at the end of the test.

Reply:
ok, I see